cfme/web_ui/form_buttons.py

```diff
@@ -122,11 +122,14 @@ def __repr__(self):
         return "{}({})".format(type(self).__name__, str(repr(self._alt)))
 
 add = FormButton("Add")
+attach_volume = FormButton('Attach', ng_click='attachClicked()')
```

Comment:
I cannot locate button by normal means: 'attach' and 'detach' buttons have no 'alt' attr, as I understand I can catch it only using ng_click here